Automattic/wp-calypso
# Use context-driven taglines for products and plans

#### Changes proposed in this Pull Request

The purpose of this PR is to give product and plans taglines a different copy depending on the site purchases. Specifically, we are introducing the following changes:

* Jetpack Free tagline changes depending on whether the site has Jetpack Backup or/and Jetpack Scan.
* Jetpack Backup and Jetpack Scan taglines change depending on whether the site has these products.

See capture below to understand the motivation behind this PR.

![image](https://user-images.githubusercontent.com/3418513/94975174-351d4800-04e7-11eb-8b95-33bd1fd0fa9e.png)

Fixes 1164141197617539-as-1194233925345529

#### Testing instructions

* Run this PR.
* Select a site with Jetpack Free.
* Make sure that Jetpack Free tagline looks like the after capture (verify with the ticket as well).

**Jetpack Free**
![image](https://user-images.githubusercontent.com/3418513/94976010-e7eea580-04e9-11eb-8621-c6c1ee9f1d99.png)

* Purchase Jetpack Backup.
* Make sure that Jetpack Free and Jetpack Backup taglines look like the after capture (verify with the ticket as well).

**Jetpack Free + Jetpack Backup**
![image](https://user-images.githubusercontent.com/3418513/94975982-ce4d5e00-04e9-11eb-8cc9-34e091d0fba1.png)

* Remove Jetpack Backup subscription and purchase Jetpack Scan.
* Make sure that Jetpack Free and Jetpack Scan taglines look like the after capture (verify with the ticket as well).

**Jetpack Free + Jetpack Scan**
![image](https://user-images.githubusercontent.com/3418513/94975937-a52ccd80-04e9-11eb-9ccc-2c954a051eba.png)

* Purchase Jetpack Backup (the site should own Jetpack Backup and Jetpack Scan)
* Make sure that Jetpack Free, Jetpack Backup, and Jetpack Scan taglines look like the after capture (verify with the ticket as well).

**Jetpack Free + Jetpack Backup + Jetpack Scan**
![image](https://user-images.githubusercontent.com/3418513/94975896-7c0c3d00-04e9-11eb-86c1-29dddf3ae9a3.png)
